pdp/static/js/canada_ex_map.js

```diff
@@ -1,5 +1,5 @@
 /*jslint browser: true, devel: true */
-/*global $, jQuery, OpenLayers, pdp, map, na4326_map_options, getBasicControls, getBoxLayer, getEditingToolbar, getHandNav, getBoxEditor, getNaBaseLayer, getOpacitySlider, Colorbar*/
+/*global $, jQuery, OpenLayers, pdp, map, na4326_map_options, getBasicControls, getBoxLayer, getEditingToolbar, getHandNav, getBoxEditor, getNaBaseLayer, getTileBaseLayer, getNa4326LiteBaseLayer, getOpacitySlider, Colorbar*/
```

Comment:
Is getTileBaseLayer used anymore?

Reply:
No, and neither is `getNa4326LiteBaseLayer`. I was thinking about keeping these in just in case we decide to use XYZ instead of WMS to get layers later down the line, but I'll see how others feel.

Reply:
Gotcha, I think you’re right to keep it just in case.